draft-ietf-quic-tls.md

```diff
@@ -602,10 +602,16 @@ HelloRetryRequest is still used for incorrect key shares.
 
 ## TLS Errors
 
-If TLS experiences an error, it MUST generate an appropriate alert
-as defined in {{TLS13}}; Section 6) and then provide it to QUIC,
-which sends the alert in a CRYPTO_CLOSE frame. All such alerts are
-"fatal" (see {{TLS13}}, Section 6.2.
+If TLS experiences an error, it generates an appropriate alert as defined in
+Section 6 of {{TLS13}}.
+
+A TLS alert is turned into a QUIC connection error by converting the alert
+description into a QUIC error code.  The alert description is added to 0x200 to
+produce a QUIC error code from the range reserved for CRYPTO_ERROR.  The
+resulting value is sent in a QUIC CONNECTION_CLOSE frame.
+
+The alert level of all TLS alerts is "fatal", a TLS stack does not generate
+alerts at the "warning" level.
```

Comment:
This is a bit confusing because stacks can in fact do that. Maybe MUST NOT?

Reply:
I thought that this was OK.  TLS 1.3 prohibits use of warning for anything but close_notify, and we don't allow TLS to close.  I guess that MUST NOT isn't a problem though.